src/System.IO.Compression.Brotli/System.IO.Compression.Brotli.csproj

```diff
@@ -12,24 +15,58 @@
     <OutputTypeEx>library</OutputTypeEx>
     <StartupObject />
   </PropertyGroup>
-  <!-- Platform specific properties -->
-  <PropertyGroup Condition="'$(Platform)' == 'amd64'">
+  <PropertyGroup Condition=" '$(Platform)' == 'x64'">
+    <SourceArch>x64</SourceArch>
+    <DefineConstants>BIT64;$(DefineConstants)</DefineConstants>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Platform)' == 'x86'">
+    
+    <SourceArch>x86</SourceArch>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(OS)' == 'Windows_NT'">
+    <SourceSystem>windows</SourceSystem>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(OS)' == 'Unix'">
+    <SourceSystem>linux</SourceSystem>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(OS)' == 'OSX'">
+    <SourceSystem>osx</SourceSystem>
+  </PropertyGroup>
+  <PropertyGroup>
+    <BrotliPath>..\..\external\Brotli\lib\$(SourceSystem)\$(SourceArch)</BrotliPath>
+  </PropertyGroup>
+  <ItemGroup>
+    <BrotliLibraries Include="$(BrotliPath)\*.*" />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetsUnix)' == 'true'">
+    <Compile Include="System\IO\Compression\Interop\Library.Unix.cs" />
```

Comment:
done. Also change logic instead of include(which didn't work use remove unnecessary libraries)